### DISCUSSION TITLE
How can I receive notifications for all new messages in a forum?

### DISCUSSION THREAD

**Discussion by jordanpadams** [pds-data-dictionaries/PDS-Data-Dictionaries.github.io#11] [Github Help] [answered]
It looks like I am only receiving notifications from this Discussion forum when I mentioned, how can I receive notifications for all conversations and comments?

**Comment by pds-ops** [comment, marked as answer]
Courtesy of @mdrum:

By default, you will be notified anytime you are mentioned, or for any discussions you participate in. For all that would like to have email notifications of all new messages in this forum:

While logged into your Github account, select "Watch":
<img width="388" alt="CleanShot 2022-03-31 at 11 37 47@2x" src="https://user-images.githubusercontent.com/93455202/161305194-95185b10-44be-483b-8329-6223000317e5.png">


and then select Custom -> check "Discussions", and Apply
<img width="335" alt="CleanShot 2022-03-31 at 11 37 55@2x" src="https://user-images.githubusercontent.com/93455202/161305117-00e82235-3559-4f12-a183-b5ddcc758b71.png">
